src/gui/tray/UserModel.cpp

```diff
@@ -643,14 +648,16 @@ Q_INVOKABLE void UserModel::openCurrentAccountLocalFolder()
 
 Q_INVOKABLE void UserModel::openCurrentAccountTalk()
 {
-    if (_users.isEmpty())
+    if (!currentUser())
         return;
 
-    QString url = _users[_currentUserId]->server(false) + "/apps/spreed";
-    if (!(url.contains("http://") || url.contains("https://"))) {
-        url = "https://" + _users[_currentUserId]->server(false) + "/apps/spreed";
+    auto talkApp = currentUser()->talkApp();
```

Comment:
nitpick: `const auto`?

I live by aggressively using const ;-)

Reply:
"aggressively using const" would be a great tattoo or sticker.